views/scss/pci-manager.scss

```diff
@@ -41,10 +51,21 @@
                 height:auto !important;
                 line-height: 1.4;
                 .switch-box{
+                    display: inline;
                     position: relative;
                     top: 5px;
                     right:7px;
                 }
+                .unregister {
```

Comment:
You don't need it, the display is correct with these 2 wrappers removed.

Reply:
This will help me keep this elements in order